### DISCUSSION TITLE
Welcome to Koriym.PhpSkeleton Discussions!

### DISCUSSION THREAD

**Discussion by koriym** [koriym/Koriym.PhpSkeleton#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

## 👋 Welcome!
  私たちはコミュニティの他のメンバーとつながるための場所としてディスカッションを使っています。私たちは、あなたが以下のようなことを望んでいます。
  * 疑問に思ったことを質問する。
  * アイデアを共有してください。
  * 他のコミュニティメンバーと交流する。
  * 他者を歓迎し、オープンマインドであること。これは、私たちが一緒に作るコミュニティであることを忘れないでください。
  一緒に作っていくコミュニティであることを忘れないでください💪。

  まずは、下記のコメント欄に自己紹介と、このコミュニティで何をするかを教えてください。

**Comment by koriym** [comment]
@77web @kenjis こんなのを開設してみました。

**Reply by kenjis** [reply]
![Screenshot](https://user-images.githubusercontent.com/87955/125191122-c3470b80-e27b-11eb-809c-1c697ade510f.png) 
タイトルの左のこの数字は何でしょうね？

**Reply by koriym** [reply]
それはRedditとかのupvoteと同じじゃないでしょうか

**Reply by 77web** [reply]
↑をおしたら増えたのでたぶんそうですね！
